grpc-sys/grpc_wrap.cc

```diff
@@ -279,10 +279,19 @@ grpcwrap_request_call_context_destroy(grpcwrap_request_call_context* ctx) {
   gpr_free(ctx);
 }
 
-GPR_EXPORT const grpc_metadata_array* GPR_CALLTYPE
-grpcwrap_batch_context_recv_initial_metadata(
-    const grpcwrap_batch_context* ctx) {
-  return &(ctx->recv_initial_metadata);
+
+GPR_EXPORT void GPR_CALLTYPE
```

Comment:
Can you separate all changes to C codes as a standalone PR? I think this part is good enough to be merged first. This is also a get around to the CI trust issue.